pyro/distributions/zero_inflated.py

```diff
@@ -17,15 +17,24 @@ class ZeroInflatedDistribution(TorchDistribution):
     :class:`ZeroInflatedPoisson` and :class:`ZeroInflatedNegativeBinomial`.
 
     :param torch.Tensor gate: probability of extra zeros given via a Bernoulli distribution.
+    :param torch.Tensor logits: logits of extra zeros given via a Bernoulli distribution.
     :param TorchDistribution base_dist: the base distribution.
     """
-    arg_constraints = {"gate": constraints.unit_interval}
+    arg_constraints = {"gate": constraints.unit_interval,
+                       "logits": constraints.real}
 
-    def __init__(self, gate, base_dist, validate_args=None):
+    def __init__(self, base_dist, gate=None, logits=None, validate_args=None):
```

Comment:
nit: can you add a `*,` so that when breaking backwards compatibility we at least do so loudly?
```diff
- gate=None, logits=None, validate_args=None):
+ gate=None, *, logits=None, validate_args=None):
```